### PR TITLE
Downscale cru405

### PR DESCRIPTION
**Pull request by kyleredilla**
This PR adds some changes to include the downscaling that was done for monthly CRU TS 4.05 temp and precip data. 

This codebase is not set up to use our best practices for environment management and code execution, and these are things that will be addressed in future PRs. So, testing is optional for this PR. If desired, you can try to get the new `snap_scripts/downscaling_v2/wrap_downscaler_cru_ts405_slurm.py` script working in your own environment using the `requirements.txt` file, and work through the new validation jupyter notebook `snap_scripts/notebooks/validate_cru_405_downscale.ipynb`. Or simply just examine the changes and comment / question / suggest. The main goal is to just get these old changes merged for continued development of this repo.

### CODE REVIEW

File: downscale/utils.py [hunk 1]
```diff
@@ -252,8 +239,10 @@ def _run_ds( d, f, operation_switch, anom=False, mask_value=0 ):
 	# set up output file metadata.
 	meta = base.meta
 	meta.update( compress='lzw' )
-	if 'transform' in meta.keys():
-		meta.pop( 'transform' )
+	# not sure why we would remove the transform?
```
Comment (charparr) [inline downscale/utils.py:242]: These are good comments!

**Comment by charparr** [comment]
I worked through the validation notebook - super nice. The only thing I noticed was the cell that  compares some time slices from the underlying CRU datasets couldn't find the `/Data/Base_Data/Climate/World/CRU_grids/CRU_TS40/cru_ts4.00.1901.2015.pre.dat.nc.gz` file - but I'm guessing that is just because of some file shuffling.

**Review by charparr — APPROVED** [review]
This is good start on owning our downscaling - I agree to just get these old changes merged for continued development of this repo.

**Comment by kyleredilla** [comment]
Shoot, that was my goof on the `cru_ts4.00.1901.2015.pre.dat.nc.gz` , I didn't see any uncompressed files due to the hyperion death and so when I uncompressed this time I removed the old .gz files not thinking and never tested that notebook with it. Thank you!

**Comment by charparr** [comment]
No worries - that was kind of my thinking! But figured I'd note it for the next go around and that it didn't need to hold up the merge.

**Pull request merged**